public/assets/js/scripts.js

```diff
@@ -146,4 +146,97 @@ $(document).ready(function () {
     });
   });
 
+  // use ajax to update select options on index page
+  var state = "#state";
+  $.ajax({
+    type: 'POST',
+    url: './includes/find-house.php',
+    data: {
+      load_state: state
```

Comment:
I don't think you need to send the state variable here. Something like this should work too.
```suggestion
      load_state: true
```